src/PlotRecipes/Potentials.jl

```diff
@@ -1,31 +1,31 @@
-function get_crosssection_idx_and_value(g::Grid{T, 3, Cylindrical}, r, φ, z)::Tuple{Symbol,Int,T} where {T <: SSDFloat}
+function get_crosssection_idx_and_value(grid::Grid{T, 3, Cylindrical}, r, φ, z)::Tuple{Symbol,Int,T} where {T <: SSDFloat}
 
     cross_section::Symbol, idx::Int = if ismissing(φ) && ismissing(r) && ismissing(z)
-        return get_crosssection_idx_and_value(g, r, T(0.0), z)
+        return get_crosssection_idx_and_value(grid, r, T(0.0), z)
     elseif !ismissing(φ) && ismissing(r) && ismissing(z)
         φ_rad::T = T(deg2rad(φ))
-        while !(g.φ.interval.left <= φ_rad <= g.φ.interval.right) && g.φ.interval.right != g.φ.interval.left
-            if φ_rad > g.φ.interval.right
-                φ_rad -= width(g.φ.interval)
-            elseif φ_rad < g.φ.interval.left
-                φ_rad += width(g.φ.interval)
+        while !(grid.φ.interval.left <= φ_rad in grid.φ.interval.right) && grid.φ.interval.right != grid.φ.interval.left
```

Comment:
This commit (edd265f) caused the bug that prevented the docs from building.
It is now hopefully fixed with 7ac770f.